### ISSUE TITLE
work command

### ISSUE THREAD

**Issue by Voidstriker** [outlaylol/economybot#5]
Hello, the work command is working just fine, but I just wanted to ask someone since I'm having problems...

How do I set cooldowns for jobs? I've been trying to figure it out but I couldn't

If you can help me I'll give you a cookie.

**Comment by Lauroline1** [comment]
Use the daily command but make it a work command
`const db = require('quick.db')
const ms = require('parse-ms')
const Discord = require('discord.js')

module.exports.run = async (client, message, args, config) => {
    let timeout = 600000 // 24 hours in milliseconds, change if you'd like.
let amount = Math.floor(Math.random() * 500) + 1; // 1-500 random number.

let daily = await db.fetch(`daily_${message.author.id}`);

if (daily !== null && timeout - (Date.now() - daily) > 0) {
    let time = ms(timeout - (Date.now() - daily));

    message.channel.send(`You are too tired to you work, you may work again in **${time.hours}h ${time.minutes}m ${time.seconds}s**!`)
} else {

let embed = new Discord.RichEmbed()
.setAuthor(`${message.author.tag}, it payed off!`, message.author.displayAvatarURL) 
.setDescription(`${message.author}, you've worked and earned ${amount}$ !`)
.setColor("RANDOM")

message.channel.send(embed)
db.add(`money_${message.author.id}`, amount)
db.set(`daily_${message.author.id}`, Date.now())
        
}}
module.exports.help = {
    name: 'work'
};
`

**Comment by TheGamer456YT** [comment]
@Lauroline1 not at all, at this line: `let timeout = 60000` you could just change it, 60000 is 24 hours in miliseconds, so just change it, and your ready to start workin'
